core/build.gradle.kts

```diff
@@ -0,0 +1,90 @@
+val kotlinVersion: String by project
+
+plugins {
+    kotlin("multiplatform") version "1.3.60-eap-76"
+}
+
+kotlin {
+    /*infra {
+        target("macosX64")
+        target("linuxX64")
+        target("mingwX64")
+    }*/
+
+    sourceSets {
+        val commonMain by getting {
+            dependencies {
+                implementation("org.jetbrains.kotlin:kotlin-stdlib-common:$kotlinVersion")
```

Comment:
We could omit version for kotlin deps and it will be inferred from the plugin version.